src/Digdir.Tool.Dialogporten.LargeDataSetGenerator/README.md

```
@@ -0,0 +1,65 @@
+    - The `Party` property is deterministically set based on a hash of the dialog ID. This `Party` is also used in the SeenLog table, which contains one entry for each dialog.
+
+
+* **Localization and Search Tags**
+    - Optionally, two wordlist files (`wordlist_en` and `wordlist_nb`) can be created for random localizations and search tags.
+    - These words are deterministically selected based on the dialog/localization ID.
+    - Each file should contain one word per line.
+    - These files are also added to `.gitignore`.
+
+#### TODO
+
+- **Parameterize Shell Scripts**
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)